### PR TITLE
refactor: increase timeouts for optgene and pathways

### PR DESCRIPTION
**Pull request by phantomas1234**

### CODE REVIEW

File: src/metabolic_ninja/tasks.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def find_pathways(product, model, max_predictions, source):
     return predictor.run(
         product,
         max_predictions=max_predictions,
-        timeout=60,
+        timeout=120,  # seconds
```
Comment (Midnighter) [inline src/metabolic_ninja/tasks.py:155]: My original intention was to permit up to one hour. I think it should be more than two minutes. Maybe 30?
```suggestion
        timeout=30 * 60,  # seconds
```

File: src/metabolic_ninja/tasks.py [hunk 2]
```diff
@@ -361,9 +361,9 @@ def opt_gene(pathway, model):
             target=pathway.product.id,
             biomass=model.biomass,
             substrate=model.carbon_source,
-            max_evaluations=1500,
+            max_evaluations=9999999999,
```
Comment (Midnighter) [inline src/metabolic_ninja/tasks.py]:
```suggestion
            max_evaluations=int(1E06),
```

**Review by Midnighter — APPROVED** [review]
Some suggestions for changes in times and evaluations. Thanks for identifying the real meaning!

File: src/metabolic_ninja/tasks.py (hunk 1 above)
Comment (phantomas1234) [reply src/metabolic_ninja/tasks.py:155]: It is 2 min per pathway, which is enough.

**Pull request merged**